### PR TITLE
cdc update docs for `cdc server` command.

### LINKED ISSUE #8741: incomplete `cdc server` command line parameters
## Error 报告

**This repository is ONLY used to solve problems related to DOCS-CN.
For other issues (TiDB, TiKV, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. What is the document directory for the problem?

https://docs.pingcap.com/tidb/stable/deploy-ticdc#description-of-ticdc-cdc-server-command-line-parameters

2. How would you like to improve it?

Incomplete `cdc server` command line parameters. Complete list as follow:
image

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

close #8741 

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by overvenus — APPROVED** [review]
LGTM

**Review by zhaoxinyu — APPROVED** [review]

File: ticdc/deploy-ticdc.md [hunk 1]
```diff
@@ -50,16 +50,16 @@ cdc server --pd=http://10.0.10.25:2379 --log-file=ticdc_3.log --addr=0.0.0.0:830
...
-- `pd`：PD client 的 URL。
-- `addr`：TiCDC 的监听地址，提供服务的 HTTP API 查询地址和 Prometheus 查询地址。
-- `advertise-addr`：TiCDC 对外访问地址。
-- `tz`：TiCDC 服务使用的时区。TiCDC 在内部转换 timestamp 等时间数据类型和向下游同步数据时使用该时区，默认为进程运行本地时区。（注意如果同时指定 `tz` 参数和 `sink-uri` 中的 `time-zone` 参数，TiCDC 进程内部使用 `tz` 指定的时区，sink 向下游执行时使用 `time-zone` 指定的时区）
-- `log-file`：TiCDC 进程运行日志的地址，默认为 `cdc.log`。
-- `log-level`：TiCDC 进程运行时默认的日志级别，默认为 `info`。
-- `ca`：TiCDC 使用的 CA 证书文件路径，PEM 格式，可选。
-- `cert`：TiCDC 使用的证书文件路径，PEM 格式，可选。
-- `key`：TiCDC 使用的证书密钥文件路径，PEM 格式，可选。
+- `addr`：TiCDC 的监听地址，提供服务的 HTTP API 查询地址和 Prometheus 查询地址。默认为 127.0.0.1:8300
```
Comment (TomShawn) [inline ticdc/deploy-ticdc.md:53]:
```suggestion
- `addr`：TiCDC 的监听地址，提供服务的 HTTP API 查询地址和 Prometheus 查询地址，默认为 `127.0.0.1:8300`。
```

File: ticdc/deploy-ticdc.md [hunk 2]
```diff
@@ -50,16 +50,16 @@ cdc server --pd=http://10.0.10.25:2379 --log-file=ticdc_3.log --addr=0.0.0.0:830
...
+- `advertise-addr`：TiCDC 对外开放地址，供客户端访问，如果未设置，等于 `addr`
```
Comment (TomShawn) [inline ticdc/deploy-ticdc.md:54]:
```suggestion
- `advertise-addr`：TiCDC 对外开放地址，供客户端访问。如果未设置该参数值，地址默认与 `addr` 相同。
```

File: ticdc/deploy-ticdc.md [hunk 3]
```diff
@@ -50,16 +50,16 @@ cdc server --pd=http://10.0.10.25:2379 --log-file=ticdc_3.log --addr=0.0.0.0:830
...
+- `pd`：TiCDC 监听的 pd 节点地址，用 ',' 来分隔多个 pd 节点地址。
```
Comment (TomShawn) [inline ticdc/deploy-ticdc.md:55]:
```suggestion
- `pd`：TiCDC 监听的 PD 节点地址，用 `,` 来分隔多个 PD 节点地址。
```

**Review by TomShawn — APPROVED** [review]
Minor comments

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**